XGBoost/README.md

```diff
@@ -0,0 +1,29 @@
+Boosting:
+
+Boosting is yet another ensemble machine learning technique which is designed to improve or boost the performance of a no. of weak learning models working together.
+
+The weak Classifier:
+
+A weak classifier is a model which can’t produce a sable result on it’s own but performs better than a random guess.
+
+Here is the Mathematical representation of a boosting classifier<br>
+                                               ![formula](https://user-images.githubusercontent.com/58946705/111885101-a7c5a580-89eb-11eb-8dee-fd71bc899a2d.jpg)
+
+Here, **T** is the no. of classifiers, **ft** is a weak learner and **x** is the data input. It returns the class
+
+ It can be depicted by the below diagram:
+ ![flowchart](https://user-images.githubusercontent.com/58946705/111885187-2a4e6500-89ec-11eb-919d-a9f260f2487c.jpg)
+
+
+### XGBoost

```

Comment:
This needs to have algorithm,advantages,disadvantages and references

Reply:
done